bc_obps/registration/migrations/0090_migrate_reg1_operation_statuses.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 5.0.13 on 2025-03-12 17:12
+
+import os
+from django.db import migrations
+import datetime
+
+"""
+One-time forward-only migration to be applied to prod data.
+Purpose: set the `status` field of Operations (created in Reg1) to match the statuses used in Administration/Registration2 modules.
+Also archives operations with a "Declined" status (only 1 as of Feb 19, 2025)
+Reg1 Status        | Reg2 Status
+-------------------+-------------------
+PENDING            | DRAFT
+APPROVED           | DRAFT
+CHANGES_REQUESTED  | DRAFT
+NOT_STARTED        | NOT_STARTED
+DRAFT              | DRAFT
+DECLINED           | Archive the operation
+"""
+
+
+def migrate_reg1_operation_statuses(apps, schema_monitor):
+    # import the required Django model
+    Operation = apps.get_model('registration', 'Operation')
+    User = apps.get_model('registration', 'User')
+
+    Operation.objects.filter(status="Pending").update(status="Draft")
+    Operation.objects.filter(status="Approved").update(status="Draft")
+    Operation.objects.filter(status="Changes Requested").update(status="Draft")
+
+    # we don't have a po_user outside of prod, so if we're not in prod, we end the migration here
+    if os.environ.get('ENVIRONMENT') != 'prod':
+        return
+
+    filtered_users = User.objects.filter(first_name="Patricia", last_name="Russell")
+
+    po_user = filtered_users.first()
+
+    Operation.objects.filter(status="Declined").update(
+        archived_by_id=po_user.user_guid, archived_at=datetime.datetime.now(datetime.timezone.utc)
+    )  # declined operations will be archived, per PR's instruction and using her user GUID
```

Comment:
We could use `set_archive` function!

Reply:
I think @andrea-williams tried that but it wasn't working for some reason. It was in a call so I can't double-check the conversation

Reply:
that sounds familiar but I can't remember exactly what the issue was